static/editor.js

```diff
@@ -350,6 +408,19 @@ define(function (require) {
         }
     };
 
+    Editor.prototype.handleThinkingGear = function (doShow, tooltipText) {
```

Comment:
This is my idea of "letting the user know we are up to something". How does it sound?